heasarc/sciserver_heasoft/Dockerfile

```diff
@@ -84,9 +84,18 @@ RUN cd /home/${sciserver_user}/heasoft-${HEASOFT_VERSION}/BUILD_DIR/ \
  && make install 2>&1 | tee /home/${sciserver_user}/install.log
 ## ------------------------------- ##
 
+## ------------------ ##
+## Apply patch 6.30.1 
+RUN cd /home/${sciserver_user} \
+ && wget https://heasarc.gsfc.nasa.gov/FTP/software/lheasoft/lheasoft6.30/updates/hdpatch_6.30.1 \
```

Comment:
Bryan found an issue with the original patch.  Did he replace it at the same version number?

Reply:
Why are you even using the patch? Just download and install the 6.30.1 tarball by specifying the HEASOFT_VERSION to be 6.30.1.

Reply:
@esabol. I am doing that. I initially thought it cannot be done. Thanks for the note.